### PR TITLE
Output JSON per jpeg

### LINKED ISSUE #20: Make a filesystem JPEG writer
image data spans over multiple platform packages. Grouping of several packages necessary . Data is encoded in jpeg.

Needs #10

### PR DESCRIPTION
**Pull request by local-minimum**
This is the last part, and thus resolves #20 

Writes a JSON with same filename (`2808949523926.png` -> `2808949523926.json`):
```json
{"origin":{"name":"/home/myname/path/MyArchive1_20200428-204613_20200428-205254.rac","processingTime":"2020-05-08T10:39:47.291542593+02:00"},"ramsesHeader":{"specification":"SPU045-S2:6F","ramsesTime":"2020-04-28T20:46:30.305Z"},"sourceHeader":{"specification":"IS-OSE-ICD-0005:1","spSequenceCount":8573},"tmHeader":{"tmHeaderTime":"1980-01-06T00:46:56.12802124Z","tmHeaderNanoseconds":2816128021240},"SID":"","RID":"CCD3","data":{"PackData":{"CCDSEL":3,"EXPTS":2808,"EXPTSS":62228,"WDW":1,"WDWOV":0,"JPEGQ":95,"FRAME":46,"NROW":250,"NRBIN":2,"NRSKIP":0,"NCOL":500,"NCBIN":4,"NCSKIP":0,"NFLUSH":1023,"TEXPMS":5000,"GAIN":0,"TEMP":4124,"FBINOV":0,"LBLNK":289,"TBLNK":302,"ZERO":121,"TIMING1":5149,"TIMING2":518,"VERSION":50,"TIMING3":41987,"NBC":0},"BadColumns":[]}}
```

**Note that:**
* No specific formatting of `CCDImage` so `PackData` and `BadColumns` instead of camelCase.
* No specific formatting of `CCDImageDataPack` which means that several complex parameters such as `EXPTS` and `EXPTSS` isn't converted to nanoseconds / date, also `WDW`, `NCBIN` and `GAIN` have complex interpretations and really should be read out better, I think. 

I'll make an issue of this unless you require it included in this iteration.

### CODE REVIEW

File: internal/exports/disk_writer.go [hunk 1]
```diff
@@ -102,14 +103,34 @@ func DiskCallbackFactory(
 				}
 
 				png.Encode(imgFile, img)
+				defer imgFile.Close()
```
Comment (joakimmoller) [inline internal/exports/disk_writer.go]: same deferred close again? already done at line 99

**Review by joakimmoller — APPROVED** [review]
OK! remove the extra Close... then I think it is fine!

File: internal/exports/disk_writer.go (hunk 1 above)
Comment (local-minimum) [inline internal/exports/disk_writer.go]:
```suggestion
```

File: internal/exports/disk_writer.go [hunk 2]
```diff
@@ -102,14 +103,34 @@ func DiskCallbackFactory(
...
+
+				// Write metadata only supports DataRecord
+				pkg, ok := expPkg.(common.DataRecord)
+				if ok {
+					ext := filepath.Ext(imgFileName)
+					jsonFileName := fmt.Sprintf(
+						"%v.json",
+						imgFileName[0:len(imgFileName)-len(ext)],
+					)
+					jsonFile, err := os.Create(jsonFileName)
```
Comment (local-minimum) [inline internal/exports/disk_writer.go:115]:
```suggestion
					jsonFile, err := os.Create(jsonFileName)
					defer jsonFile.Close()
```

File: internal/exports/disk_writer.go (hunk 1 above)
Comment (local-minimum) [reply internal/exports/disk_writer.go]: fixed

**Pull request merged**